recipe/meta.yaml

```diff
@@ -135,7 +136,7 @@ outputs:
         - {{ pin_subpackage('gap-defaults', exact=True) }}
     test:
       commands:
-        - echo "TODO - test a package here"
+        - echo 'if LoadPackage("alnuth") = fail then QUIT_GAP(1); else QUIT_GAP(0); fi;' | gap
 
 about:
```

Comment:
The `doc_url` below could be changed to https://docs.gap-system.org/doc/ref/chap0_mj.html